### PR TITLE
fix: project logic

### PR DESCRIPTION
**Pull request by naomi-lgbt**
Checklist:

- [ ] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [ ] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #XXXXX

Need a live branch to test against - will mark ready for review when debugged + cleaned up

### CODE REVIEW

File: src/plugins/hide-curriculum-strings.ts [hunk 1]
```diff
@@ -38,7 +38,7 @@ const createChallengeTitleLookup = (
       },
     };
   } catch (err) {
-    console.log(JSON.stringify(err), null, 2);
+    console.log(JSON.stringify(err, null, 2));
```
Comment (ShaunSHamilton) [inline src/plugins/hide-curriculum-strings.ts:41]: This one is quite funny...

**Review by ShaunSHamilton — APPROVED** [review]
Maybe I should stop approving these 😟

File: src/plugins/hide-curriculum-strings.ts (hunk 1 above)
Comment (nhcarrigan) [reply src/plugins/hide-curriculum-strings.ts:41]: It took me far too long to figure out why `null 2` was printing.

**Comment by nhcarrigan** [comment]
Hey I tested this one it works

**Pull request merged**